### PR TITLE
Fix "Build a generic plot" processing algorithm

### LINKED ISSUE #234: [Processing] Build a generic plot
Hi @ghtmtt,
when I try to do Build a generic plot using processing toolbox I have this Python error:

```python

2020-08-07T13:33:49     WARNING    Traceback (most recent call last):
              File "C:/OSGEO4~1/apps/qgis-ltr/./python/plugins\processing\gui\ProcessingToolbox.py", line 236, in executeAlgorithm
              alg = self.algorithmTree.selectedAlgorithm().create() if self.algorithmTree.selectedAlgorithm() is not None else None
             _core.QgsProcessingException: Error creating algorithm from createInstance()

```

Tested on a new profile only with plotly plugin install.

QGIS version | 3.10.6-A Coruña | QGIS code revision | ec80021f49
-- | -- | -- | --
Compiled against Qt | 5.11.2 | Running against Qt | 5.11.2
Compiled against GDAL/OGR | 3.0.4 | Running against GDAL/OGR | 3.0.4
Compiled against GEOS | 3.8.1-CAPI-1.13.3 | Running against GEOS | 3.8.1-CAPI-1.13.3
Compiled against SQLite | 3.29.0 | Running against SQLite | 3.29.0
PostgreSQL Client Version | 11.5 | SpatiaLite Version | 4.3.0
QWT Version | 6.1.3 | QScintilla2 Version | 2.10.8
Compiled against PROJ | 6.3.2 | Running against PROJ | Rel. 6.3.2, May 1st, 2020
OS Version | Windows 7 SP 1 (6.1)
Active python plugins | DataPlotly; db_manager; MetaSearch; processing

Plotly QGIS python version: 3.3.0

Cheers.

### PR DESCRIPTION
**Pull request by agiudiceandrea**
Fixes #234.

### CODE REVIEW

File: DataPlotly/processing/dataplotly_algorithms.py [hunk 1]
```diff
@@ -83,6 +83,9 @@ def tr(string, context=''):
             context = 'Processing'
         return QCoreApplication.translate(context, string)
 
+    def createInstance(self):
```
Comment (ghtmtt) [inline DataPlotly/processing/dataplotly_algorithms.py:86]: can I suggest to use the following?
```python
def createInstance(self):
    return DataPlotlyProcessingPlot()
```

**Review by ghtmtt — CHANGES_REQUESTED** [review]
thanks for taking care of this @agiudiceandrea :) even if this fix let the UI opens, then there are other issues related to the dataplotly factory...

File: DataPlotly/processing/dataplotly_algorithms.py (hunk 1 above)
Comment (agiudiceandrea) [reply DataPlotly/processing/dataplotly_algorithms.py:86]: OK, it is the same...

File: DataPlotly/processing/dataplotly_algorithms.py [hunk 2]
```diff
@@ -83,6 +83,9 @@ def tr(string, context=''):
...
+        return type(self)()
```
Comment (agiudiceandrea) [inline DataPlotly/processing/dataplotly_algorithms.py:87]:
```suggestion
        return DataPlotlyProcessingPlot()
```

**Review by ghtmtt — APPROVED** [review]
perfect thanks!

**Review by ghtmtt — APPROVED** [review]

**Comment by ghtmtt** [comment]
thanks @agiudiceandrea

**Pull request merged**